.circleci/config.yml

```diff
@@ -439,8 +372,73 @@ jobs:
       - store_artifacts:
           path: /tmp/license-scanner/results
           prefix: licenses
+
+  image-scan:
+    executor: anchore/anchore_engine
+    steps:
+      - setup_remote_docker
+      - checkout
+      - attach_workspace:
+          at: /tmp
+      - run:
+          name: Load the pre-built docker image from workspace
+          command: docker load -i /tmp/docker-image.tar
+      - anchore/analyze_local_image:
+          dockerfile_path: ./Dockerfile
+          image_name: ${DOCKER_ORG}/${CIRCLE_PROJECT_REPONAME}:${CIRCLE_TAG}
+          policy_bundle_file_path: .circleci/.anchore/policy_bundle.json
+          # Anchore bug: if policy_failure is `true`, reports don't get written - we manually check for 
+          policy_failure: false
+          timeout: '500'
+      - run:
+          name: Evaluate Failures.
+          command: |
+            if [[ ! $(which jq) ]]; then
+              (set +o pipefail; apk add jq || apt-get install -y jq || yum install -y jq)
+            fi
+            if [[ $(ls anchore-reports/*content-os*.json 2> /dev/null) ]]; then
+              printf "\n%s\n" "The following OS packages are installed:"
+              jq '[.content | sort_by(.package) | .[] | {package: .package, version: .version}]' anchore-reports/*content-os*.json
+            fi
+            if [[ $(ls anchore-reports/*vuln*.json 2> /dev/null) ]]; then
+              printf "\n%s\n" "The following vulnerabilities were found:"
+              jq '[.vulnerabilities | group_by(.package) | .[] | {package: .[0].package, vuln: [.[].vuln]}]' anchore-reports/*vuln*.json
+            fi
+
+            # Check the status of the policy.json for whether or not our tests failed
+            failCount=$(cat anchore-reports/*policy*.json | grep '\"status\": \"fail\"' | wc -l)
+            if [ $failCount -gt 0 ]; then
+              echo "Failed with a policy failure count of: ${failCount}"
+              # TODO: reenable when we are happy with testing
+              # exit 1
+            fi
+
+      - store_artifacts:
+          path: anchore-reports
+
+  publish:
+    executor: default-machine
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp
       - run:
-          <<: *defaults_build_docker_build_release
```

Comment:
Instead of rebuilding the image from scratch, I've replaced this step by simply re-tagging the existing image for release. This saves time, and also gives up a guarantee that the image we scanned is exactly the release image as well.